openghg/standardise/boundary_conditions/_boundary_conditions.py

```diff
@@ -0,0 +1,75 @@
+import logging
+from pathlib import Path
+from typing import Dict, Optional, Union
+import xarray as xr
+
+from openghg.util import clean_string, timestamp_now, synonyms
+from openghg.store import update_zero_dim
+
+logger = logging.getLogger("openghg.standardise.boundary_conditions")
+logger.setLevel(logging.DEBUG)  # Have to set level for logger as well as handler
+
+
+def parse_boundary_conditions(
```

Comment:
The way we've done this with other data types is to call the `parse_*` function which is basically just our internal format `parse_openghg`. So to be consistent with the other data types, this should be `parse_openghg` and the source_format should be "openghg".